sklearn/linear_model/sag.py

```diff
@@ -170,6 +170,8 @@ def sag_solver(X, y, sample_weight=None, loss='log', alpha=1.,
     Ridge, SGDRegressor, ElasticNet, Lasso, SVR, and
     LogisticRegression, SGDClassifier, LinearSVC, Perceptron
     """
+    if not warm_start_mem:
+        warm_start_mem = dict()
```

Comment:
There's no difference, effectively, but because you insisted on it elsewhere, I thought it strange that you should be inconsistent.